platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -87,6 +87,12 @@ The information comes from `@since` tags in the Javadoc, but which weren't displ
 Using the new `--no-comments` option allows generating projects that contain only code without clarifying comments.
 The resulting build files and source files are smaller and less noisy.
 
+#### Emitting separate problems for each issue found in a TOML catalog file
+
+When a TOML catalog files are used with invalid TOML syntax every distinct problem is emitted separately via the Problems API.
```

Comment:
```suggestion
When TOML catalog files with invalid TOML syntax are used, each distinct problem is reported separately using the Problems API.
```

Would it be possible to include a screenshot or a snapshot of the report?

Reply:
It's just an even we send, what it looks like in a report entirely depends on the consumer e.g. the IDEs.

Reply:
Sounds good, no screenshot, just the grammar update and we are good to go.